src/Microsoft.Windows.CsWin32/Microsoft.Windows.CsWin32.csproj

```diff
@@ -48,6 +48,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.8.0" />
     <PackageReference Include="Nullable" Version="1.3.0" />
+    <PackageReference Include="IsExternalInit" Version="1.0.0" />
```

Comment:
Please revert the `IsExternalInit` change. That package is buggy.
https://github.com/manuelroemer/IsExternalInit/issues/5